docs/firstobject.md

```diff
@@ -76,78 +76,84 @@ There is of course a lot more to Nodes, Content Types, Formatters, Widgets and i
+
+![Ingest Step 4 - Attach Files pt 2](../imgs/ingest-first-object_beta2/09_step-four_upload-files_2.jpg)
 
-This step is pretty straight forward: Click `Choose Files` to open your file selector window and choose which file you would like to ingest. After you've done this, click `Save Metadata`. 
+Once you've mentally digested all of that data, let's go ahead and click `Save Metadata`.
 
-Two things to note: The `Save Metadata` button simply persists the metadata in the current webform session. The actual ingest of the Object happens when you click `Save` (see the **Complete** step below). Secondly, each "upload field" has a set of allowed file types (such as .jpg, .tiff, .png) and in this demo there are also file size restrictions which are explained in the Upload Field's description.
+**Question:** "Wait! Don't I need to switch `Save as` from *Draft* to *Publish*? I'm ready for this Digital Object to be shared with the world!"
 
-From here, Steps 5 & 6 (hotspots & Preview) are skipped and you are directed to the **Complete** step.
+**Answer:** By clicking `Save Metadata` we are simply persisting all the metadata in the current webform session. The actual *ingest* of the Object happens when you click `Save` on the next and final step, `Complete`.
 
 ### Complete
 
-![Ingest Step7](../imgs/ingest-step-7_Complete.jpg)
+Alright, we've made it. We've added metadata, linked Data, uploaded our files and now... we're ready to **save**! Go ahead and change the status from *Draft* to *Published* and click `Save`.
 
-You made it! You added your Metadata, Linked Data, Uploaded your files and now you're ready to save! First, give your object a preview by clicking the `Preview` button directly to the right of the `Save` button.
+![Ingest Step 7 - Complete 1](../imgs/ingest-first-object_beta2/10_step-six_complete.jpg)
 
-Does everything look alright? Great! Click the `Back to content editing` button in the top left corner to go through with saving your object.
+Once you hit save you should see the following green messages and your first Archipelago Digital Object!
 
-![Preview-page-screenshot](../imgs/ingest-documentation_preview-page.png)
+![Ingest Step 7 - Complete Green Messages](../imgs/ingest-first-object_beta2/11_step-six_complete_green-messages.jpg)
 
-Now you are back to where you began, and you are ready to save your First Digital Object! **Be sure to switch your "Save As" status from *Draft* to *Publish*.** Once you hit `Save` you should see a few green push notifications indicating your object has been created. 
+![Ingest Step 7 - Complete 2](../imgs/ingest-first-object_beta2/12_step-six_complete_2.jpg)
 
-Like this:
-![Ingest Step7 Saved](../imgs/ingest-step-7_Complete-save.jpg)
+Congratulations on creating your first digital object! 🍓
 
-Congratulations on creating your first digital object!
+➡️[Back to Instructions and Guides](https://github.com/esmero/archipelago-documentation#archipelago-deployment-quickstart)
```

Comment:
Please remove the arrow icon. Confusing to have a Back pointing forward. Just the link feels cleaner, arrow can be replaced if you want so for << or something like that. Thx